src/components/Datetime.tsx

```diff
@@ -46,7 +46,11 @@ export default function Datetime({
 }
 
 const FormattedDatetime = ({ pubDatetime, modDatetime }: DatetimesProps) => {
-  const myDatetime = new Date(modDatetime ? modDatetime : pubDatetime);
+  const myDatetime = new Date(
+    (modDatetime ?? pubDatetime) > pubDatetime
+      ? modDatetime ?? pubDatetime
+      : pubDatetime
+  );
```

Comment:
Instead of this, how about something like this

```tsx
  const myDatetime = new Date(
    modDatetime && modDatetime > pubDatetime ? modDatetime : pubDatetime
  );
```

I think it is more concise and easy to understand.